.ci/install-sdk.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+DEST="${GOPATH}/bin/operator-sdk"
+
+function install_sdk() {
+    echo "Downloading the operator-sdk ${SDK_VERSION} into ${DEST}"
+    curl https://github.com/operator-framework/operator-sdk/releases/download/${SDK_VERSION}/operator-sdk-${SDK_VERSION}-x86_64-linux-gnu -sLo ${DEST}
+    chmod +x ${DEST}
+}
+
+if [ ! -d ${GOPATH}/bin/ ]; then
```

Comment:
`mkdir -p` is noop and does not fail if the dir exists. The check does not seem to be necessary.